tls.c

```diff
@@ -107,11 +107,19 @@ static bool load_server_certificates(char **errmsg) {
     } else if (!SSL_CTX_check_private_key(settings.ssl_ctx)) {
         snprintf(error_msg, errmax, "Error validating the certificate\r\n");
         success = false;
-    } else {
-        settings.ssl_last_cert_refresh_time = current_time;
+    } else if (settings.ssl_ca_cert) {
```

Comment:
This is repetitive from https://github.com/memcached/memcached/blob/master/tls.c#L155. Let's remove loading the  CA cert from there and keep it only here, because this method is used in the `ssl_init` as well.